app/views/modal-more-info.html

```diff
@@ -6,18 +6,18 @@ <h4 class="modal-title" id="myModalLabel">Info</h4>
   <p>Ben je klaar om bezoekers met UiTPAS of Paspartoe te ontvangen?</p>
   <p>Kijk dan na of je onderstaande stappen goed gevolgd hebt:</p>
   <strong>Maak een UiTiD aan</strong>
-  <p>Hiermee kan je inloggen op <a href="http://beheer.UiTPAS.be">beheer.UiTPAS.be</a>, <a href="http://www.uitdatabank.be">www.UiTdatabank.be</a> en <a href="http://www.uitpas.be">www.UiTPAS.be</a></p>
-  <strong>Stuur het e-mail adres van jouw UiTiD en de naam van de vereniging waarvoor je je wil aanmelden naar <a href="mailto:vragen@UiTdatabank.be">vragen@UiTdatabank.be</a></strong>
-  <p>Wij koppelen jouw UiTiD aan de juiste organisatie op beheer.UiTPAS.be</p>
-  <strong>Voer jouw activiteiten in via <a href="http://www.uitdatabank.be">www.UiTdatabank.be</a></strong>
-  <p>Je kan inloggen met jouw UiTiD. Voeg alle UiTPASactiviteiten toe en vergeet zeker niet de organisator te selecteren. Voeg de UiTPASvelden toe.</p>
-  <strong>Controleer of jouw activiteit verschijnt via <a href="http://beheer.UiTPAS.be">beheer.UiTPAS.be</a></strong>
-  <p>Dit kan je nakijken vanaf één dag na invoer van je activiteit op <a href="http://www.uitdatabank.be">www.UiTdatabank.be</a></p>
-  <strong>Zorg voor een UiTPAS-scanner (gsm of mini-pc)</strong>
-  <p>Op de meeste locaties waar activiteiten plaatsvinden is er een UiTPASscanner aanwezig. Vraag dit best na op voorhand. Woon je in regio Aalst en wil je een scanner ontlenen, klik dan hier voor een overzicht van de uitleendiensten.</p>
-  <strong>Meld je aan op de gsm of mini-pc om punten te sparen en kansentarief toe te kennen</strong>
-  <p>Aanmelden doe je op de gsm of op de laptop via het UiTPAS-icoontje op het bureaublad. Controleer of het apparaat verbonden is met het internet.</p>
-  <p>Heb je nog vragen? Contacteer ons via jouwgemeente@uitpas.be</p>
+  <p>Hiermee kan je inloggen op <a href="http://balie.uitpas.be">balie.uitpas.be</a>, <a href="http://www.uitdatabank.be">www.uitdatabank.be</a> en <a href="http://www.uitpas.be">www.uitpas.be</a></p>
```

Comment:
Als protocol gebruik je beter ook altijd `https://`
bv.
```suggestion
  <p>Hiermee kan je inloggen op <a href="https://balie.uitpas.be">balie.uitpas.be</a>, <a href="https://www.uitdatabank.be">www.uitdatabank.be</a> en <a href="https://www.uitpas.be">www.uitpas.be</a></p>
```